hera_cal/noise.py

```diff
@@ -34,6 +35,37 @@ def interleaved_noise_variance_estimate(vis, kernel=[[1, -2, 1], [-2, 4, -2], [1
     return variance
 
 
+def infer_dt(times_by_bl, bl, default_dt=None):
+    '''Attemps to infer dt for a baseline from times by bl. If len(times_by_bl[bl])
+    
+    Arguments:
```

Comment:
Maybe consider adopting the numpy format which is prevalent in many docstrings in `hera_cal`

Reply:
I can see this going either way though since `redcal.py` does not use this convention. Up to you.

Reply:
I'm going to keep this consistent within this module. If we want to make the whole repo one style, that feels like a separate PR.

Reply:
👍